packages/wallet/src/__stories__/dummy-wallet-states.ts

```diff
@@ -14,24 +14,26 @@ export const defaultParams = {
   uid: "uid",
   commitments: [
     {commitment: preFundCommitment1, signature: "fake-sig"},
-    {commitment: preFundCommitment2, signature: "fake-sig"}
+    {commitment: preFundCommitment2, signature: "fake-sig"},
   ],
   turnNum: preFundCommitment2.turnNum,
   networkId: 3,
   challengeExpiry: 0,
   transactionHash: "0x0",
   userAddress: "0x0",
-  funded: false
+  funded: false,
 };
 
 ////////////////////////////
 // WALLET NOT INITIALIZED //
 ////////////////////////////
 
 export const dummyWaitForLogin: walletStates.WalletState = walletStates.waitForLogin();
-export const dummyWaitForMetaMask: walletStates.WalletState = walletStates.metaMaskError({
-  ...dummyWaitForLogin
-});
+export const dummyWaitForMetaMask: walletStates.WalletState = walletStates.metaMaskError(
+  {
+    ...dummyWaitForLogin,
```

Comment:
It becomes difficult to manage PRs that are merged _after_ a change to prettier configuration:
- I might have significantly changed the AST in `my-branch`
- You merge `your-branch`, which has a lot of LOC changed due to `prettier:write`
- I have to spend a lot of time resolving merge conflicts.

If you contain the changes to prettier configuration into its own PR, with no other changes, then the converse is easy:
- I might have significantly changed the AST in `my-branch`
- You change prettier config in `your-branch` in commit `C1`
- You run `yarn prettier:write` in commit `C2`
- You notice that `my-branch` was merged into master, and when you write `git rebase master`, you have merge conflicts in `your-branch`

Resolving your merge conflicts is easy: `git reset --hard C1 && yarn prettier:write && git add -a && git commit -, $MESSAGE`

Because of this, we use the convention that anytime a tool is used to automatically modify code,
1. those changes are committed in a single commit
2. it's contained in the smallest PR possible
3. once there are no open PRs, we resolve merge conflicts in the commit where the code was modified, and merge it.

Reply:
In this case, I'm not intending to edit any prettier configurations at all. When I ran `yarn build` I hit `tslint` issues (specifically, `trailing-comma` was not respected). So, to fix that, I ran `tslint --fix`.